modules/example/src/smithy4s/example/PersonAge.scala

```diff
@@ -0,0 +1,13 @@
+package smithy4s.example
+
+import smithy4s.Newtype
+import smithy4s.schema.Schema._
+
+object PersonAge extends Newtype[Int] {
```

Comment:
Ah I did have this working where it was Newtype[Age] earlier, but I must have messed it up with another change I made. I'll resolve tomorrow.